datasets/thainer/thainer.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+_CITATION = """\
+@misc{Wannaphong Phatthiyaphaibun_2019,
+    title={wannaphongcom/thai-ner: ThaiNER 1.3},
+    url={https://zenodo.org/record/3550546},
+    DOI={10.5281/ZENODO.3550546},
+    abstractNote={Thai Named Entity Recognition},
+    publisher={Zenodo},
+    author={Wannaphong Phatthiyaphaibun},
+    year={2019},
+    month={Nov}
+}
+"""
+
+_LICENSE = "CC-BY 3.0"
+
+_DESCRIPTION = """\
+ThaiNER (v1.3) is a 6,456-sentence named entity recognition dataset created from expanding the 2,258-sentence
+[unnamed dataset](http://pioneer.chula.ac.th/~awirote/Data-Nutcha.zip) by
+[Tirasaroj and Aroonmanakun (2012)](http://pioneer.chula.ac.th/~awirote/publications/).
+It is used to train NER taggers in [PyThaiNLP](https://github.com/PyThaiNLP/pythainlp).
+The NER tags are annotated by [Tirasaroj and Aroonmanakun (2012)]((http://pioneer.chula.ac.th/~awirote/publications/))
+for 2,258 sentences and the rest by [@wannaphong](https://github.com/wannaphong/).
+The POS tags are done by [PyThaiNLP](https://github.com/PyThaiNLP/pythainlp)'s `perceptron` engine trained on `orchid_ud`.
+[@wannaphong](https://github.com/wannaphong/) is now the only maintainer of this dataset.
+"""
+
+
+class ThaiNerConfig(datasets.BuilderConfig):
+    """BuilderConfig for ThaiNer."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for ThaiNer.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(ThaiNerConfig, self).__init__(**kwargs)
+
+
+class ThaiNer(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Thainer(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `thainer`